cmd/migrate/README.md

```diff
@@ -33,6 +33,13 @@ $ apt-get update
 $ apt-get install -y migrate
 ```
 
+### Dockerfile
+
+```dockerfile
+# https://hub.docker.com/r/migrate/migrate/tags
+COPY --from=migrate/migrate:$version /usr/local/bin/migrate /usr/local/bin/migrate
```

Comment:
This might not work since we don't provide static builds in the Dockerfile. You're better off using the Go toolchain build instructions in your Dockerfile to avoid this issue.